app/views/reports/elements/_my_module_repository_element.html.erb

```diff
@@ -1,17 +1,8 @@
-<% my_module ||= @my_module %>
-<% element_id ||= repository&.id %>
+<% repository ||= report_element.repository %>
```

Comment:
For deleted repositories, `report_element` will be `nil`. We pass only `repository_snapshot`

Reply:
What? Why would you even create report element for deleted inventory?

Reply:
I mean if my_module has a snapshot from a deleted repository. Repository itself not exist. But we pass snapshot.

Reply:
Report element should have connection to the repository, even if it is a snapshot

Reply:
Then we need to introduce a new report element - repoistory_snapshot. And create it for each my_module.